integration/shared/isolated/add_network_policy_command_test.go

```diff
@@ -148,5 +186,18 @@ var _ = Describe("add-network-policy command", func() {
 				Eventually(session).Should(Exit(1))
 			})
 		})
+
+		When("policy to another space is specified", func() {
+			When("org is specified but space is not", func() {
+				It("returns an error", func() {
+					session := helpers.CF("add-network-policy", appName, "--destination-app", appName, "-o", "myorg")
+
+					Eventually(session.Err).Should(Say("Incorrect Usage: space must be provided with org, the required flag '-s' was not specified"))
+					Eventually(session).Should(Say("FAILED"))
+					Eventually(session).Should(Say("NAME:"))
```

Comment:
Can you replace "NAME:" with "USAGE". It will be much clear that the test is looking for a help text.

Reply:
Done.